samples/online-boutique/kubernetes-manifests/deployments/checkoutservice.yaml

```diff
@@ -35,15 +35,22 @@ spec:
       serviceAccountName: checkout
       containers:
         - name: server
-          image: gcr.io/google-samples/microservices-demo/checkoutservice:v0.3.6
+          securityContext:
+            allowPrivilegeEscalation: false
+            capabilities:
+              drop:
+                - ALL
+            privileged: false
+            readOnlyRootFilesystem: true
```

Comment:
These `securityContext` came from https://github.com/GoogleCloudPlatform/microservices-demo/pull/887.
We limited the privileges of all microservice containers.